### ISSUE TITLE
ship doesn't boot after vere upgrade

### ISSUE THREAD

**Issue by Quodss** [urbit/vere#732]
After vere upgrade on edge, event log replay is followed by some newt activity, followed by pier:EOF error and crash.

vere version: `urbit 3.1-98bee2a07c`

**Comment by Quodss** [comment]
My previous upgrade happened on October 7th, so the bug was introduced on 8th or 9th

**Comment by Quodss** [comment]
Attempting to boot a comet with that binary causes it to throw `Illegal instruction (core dumped)`

Fakeship boot reaches metamorphosis followed by pier:EOF

**Comment by pkova** [comment]
Is this Linux x86-64 or something else?

**Comment by Quodss** [comment]
Linux x86-64

**Comment by pkova** [comment]
We discovered out of band that @Quodss is running on a Native Planet Tellurian 1 that's running Intel Celeron Apollo Lake, a chip with no support for AVX2. @mopfel-winrux has provided us with access to a Tellurian and we were able to reproduce the problem. 

It's unclear why our build produces different SIMD instructions than before, but I see that the openssl build compiles an assembly file called `rsaz-avx2.s`. This in combination with the fact that we are unable to reproduce this on a Tellurian 2 or any other chip that supports AVX2 is some serious circumstantial evidence. We'll fix the edge build tomorrow.

**Comment by mopfel-winrux** [comment]
Just some corrections. hes running on a Tellurian 2 with an intel apollo lake chip. It doesn't have avx support. You have access to the same chip.

The new vere works on a tellurian 1 which is an intel i5 and does support an AVX2

**Comment by pkova** [comment]
The error ended up being a misconfigured GMP build that was mistakenly set to optimize for the specific chipset that happened to run the `/.configure` step instead of a more general x86_64 target. The resulting binary had a %mulx instruction which was not supported on the older NP hardware (feature flags BMI2 and AVX):
```
(gdb) display/i $pc
1: x/i $pc
=> 0x134c87b <__gmpn_mul_1+155>:	mulx   (%rsi),%r9,%rax
```

This is fixed by #733, @Quodss please report back after letting [the CI job](https://github.com/urbit/vere/actions/runs/11278632043/job/31367532041) run and running `urbit next` on `edge`.

**Comment by Quodss** [comment]
upgraded to 218e8a16ce, looks good
